### PR TITLE
Added AioControllerGetByName to allow retrieval of Aio Controller by

### PR DESCRIPTION
**Pull request by PWAlessi**
Added AioControllerGetByName to allow retrieval of Aio Controller by name

Signed-off-by: PWAlessi <pwalessi1@gmail.com>

### CODE REVIEW

File: storage/v1/backend_aio.proto [hunk 1]
```diff
@@ -15,6 +15,7 @@ service AioControllerService {
     rpc AioControllerCreate (AioControllerCreateRequest) returns (AioController) {}
     rpc AioControllerDelete (AioControllerDeleteRequest) returns (google.protobuf.Empty) {}
     rpc AioControllerGet    (AioControllerGetRequest)    returns (AioController)    {}
+    rpc AioControllerGetByName    (AioControllerGetByNameRequest)    returns (AioController)    {}
```
Comment (glimchb) [inline storage/v1/backend_aio.proto:18]: how `AioControllerGet` and `AioControllerGetByName` are different ? 
commit message doesn't explain
also why do we need it ? how one would use it... ?

Comment (PWAlessi) [reply storage/v1/backend_aio.proto:18]: I can get rid of it. We can always add it back in later if we want to.

File: storage/v1/backend_aio.proto [hunk 2]
```diff
@@ -30,6 +30,7 @@ message AioController {
     int64 block_size = 4;
     int64 num_blocks = 5;
     common.v1.Uuid uuid =6;
+    string filename = 7;
```
Comment (glimchb) [inline storage/v1/backend_aio.proto:33]: also looks like `int64 id = 2;` is redundant...

Comment (PWAlessi) [reply storage/v1/backend_aio.proto:33]: Ok, I'll remove it

**Review by glimchb — APPROVED** [review]
lgtm

**Pull request merged**